src/PKSim.Presentation/DTO/Parameters/ParameterDTO.cs

```diff
@@ -78,8 +77,13 @@ public virtual double Value
       {
          get
          {
-            var (result,_) = Parameter.TryGetValueInDisplayUnit();
+            var (result, success) = Parameter.TryGetValueInDisplayUnit();
             //Permeability cannot be read in compound as references cannot be resolved
+            //We return zero for now so that the rules are not showing a broken state (>=0). 
+            //We probably need to update the rules framework to not evaluate on NaN.
+            if (!success)
+               return 0;
```

Comment:
This was returnin NaN. But in fact, we have a rule that the value should be 0 or greater and this was breaking the rule
So returning 0 for now until we make it more resiliehnt